docs/src/main/paradox/bootstrap/index.md

```diff
@@ -86,6 +86,71 @@ is strictly defined and is as follows:
   to be using the Bootstrap mechanism.
 @@@
 
+## Deployment considerations
+
+### Initial deployment
+
+Cluster Bootstrap will always attempt to join an existing cluster if possible. However if no other contact point advertises any `seed-nodes` a
+new cluster will be formed by the node decided by the `JoinDecider` where the default sorts the addresses then picks the lowest.
+
+A setting is provided, `akka.management.cluster.bootstrap.new-cluster-enabled` that can be disable new cluster formation to only allow the
+node to join existing clusters. 
+
+* On initial deployment use the default `akka.management.cluster.bootstrap.new-cluster-enabled=on`
+* Following the initial deployment it is recommended to set `akka.management.cluster.bootstrap.new-cluster-enabled=off` 
+  with an immediate re-deployment once the initial cluster has formed
+  
+This can be used to provide additional safety during restarts and redeploys while
+there is a network partition present. Without new cluster formation disabled an isolated set of nodes could form a new
+cluster if all are restarted. 
+
+### Recommended Configuration
+
+When using the bootstrap module, there are some underlying Akka Cluster settings that should be specified to ensure
+that your deployment is robust.
+
+Since the target environments for this module are dynamic, that is, instances can come and go, failure needs to be
+considered. The following configuration will result in your application being shut down after 30 seconds if it is unable to
+join the discovered seed nodes. In this case, the orchestrator (i.e. Kubernetes or Marathon) will restart your node
+and the operation will (presumably) eventually succeed. You'll want to specify the following in your `application.conf` file:
+
+@@snip [CompileOnly.scala](/bootstrap-demo/local/src/main/resources/application.conf) { #coorindated-shutdown }
+
+### Rolling updates
+
+#### Graceful shutdown 
+
+Akka Cluster can handle hard failures using a downing provider such as Lightbend's split brain resolver discussed below.
+However this should not be relied upon for regular rolling redeploys. Features such as `ClusterSinglton`s and `ClusterSharding`
+can safely restart actors on new nodes far quicker when it is certain that a node has shutdown rather than crashed. 
+
+Graceful leaving will happen with the default settings as it is part of Coordinated Shutdown. Just ensure that
+a node is sent a `SIGTERM` and not a `SIGKILL`. Environments such as Kubernetes will do this, it is important to ensure 
+that if JVM is wrapped with a script that it forwards the signal. Upon receiving a `SIGTERM` Coordinated shutdown will:
+
+* Perform a `Cluster(system).leave` on its self
+* The `Leader` will then move the node to `Exiting` and then `Removed` allowing any shards to be shutdown gracefully
```

Comment:
I don't think we need to bother the user with introducing the concept of leader here.

The status of the member will be changed to `Exiting` while allowing any shards to be shutdown gracefully and `ClusterSingleton`s to be migrated if this was the oldest node. Finally the node is removed from the Akka Cluster membership.